rasa/core/featurizers.py

```diff
@@ -376,6 +452,27 @@ def _featurize_labels(
 
         return y
 
+    def _postprocess_trackers_as_states(
```

Comment:
is it simpler to check `if PREV_PREFIX + ACTION_LISTEN_NAME in state:` otherwise remove intent?